src/System.Device.Gpio.Tests/GpioControllerTestBase.cs

```diff
@@ -116,12 +116,12 @@ public void ThrowsIfWritingOnInputPin()
         }
 
         [Fact]
-        public void ThrowsIfReadingFromOutputPin()
+        public void CanReadFromOutputPin()
         {
             using (GpioController controller = new GpioController(GetTestNumberingScheme(), GetTestDriver()))
             {
                 controller.OpenPin(OutputPin, PinMode.Output);
-                Assert.Throws<InvalidOperationException>(() => controller.Read(OutputPin));
+                Assert.Equal(PinValue.Low, controller.Read(OutputPin));
```

Comment:
seems this is incidental that this value is "Low". On Windows it seems to be "High", please pass value to OpenPin to get stable behavior. cc: @joperezr (double check this is not a bug)

Reply:
:sweat_smile: Yep, forgot about Windows and Linux not doing the same thing here. I believe it's expected behavior, and there's already a comment about a similar side-effect [on line 21](https://github.com/dotnet/iot/blob/master/src/System.Device.Gpio.Tests/GpioControllerTestBase.cs#L21).

Reply:
Perhaps something we can iterate on separately if anyone complains